app/components/nextflow/samplesheet/metadata_cell_component.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Nextflow
+  module Samplesheet
+    # Component to render a cell with sample metadata into the sample sheet
+    class MetadataCellComponent < ViewComponent::Base
+      attr_reader :form, :metadata, :name
+
+      def initialize(form:, name:, sample:, entry:)
+        @form = form
+        @name = name
+        @metadata = if entry['meta'].is_a?(Array)
```

Comment:
**nitpick (blocking):** I think you should just select the first item that matches. Joining by ',' will cause the resulting `samplesheet.csv` to be broken as we are not quoting the values in it so that row would have more than the specified number of columns.

Reply:
Updated in f30bebcda627b08b2d3154dd820024eea6fcaf8b